v2/emailpassword/pre-built-ui/enable-email-verification.mdx

```diff
@@ -137,17 +137,17 @@ SuperTokens.init({
 
 <TabItem value="angular">
 
```

Comment:
the code snippets for angular and vue are essentially identical here.. just remove the angular and vue tabs and the content will get automatically replicated from the reactjs tab. 

This applies to all other code snippets changes in the PR as well wherein the angular and vue is the same as react. Also, you don't need to have the `title="/app/app.component.ts "` stuff anymore for angular and vue cause there are no two inits.